src/main/scala/com/cognite/sdk/scala/common/OAuth2.scala

```diff
@@ -65,106 +153,52 @@ object OAuth2 {
   object ClientCredentialsProvider {
     def apply[F[_]](
         credentials: ClientCredentials,
-        refreshSecondsBeforeTTL: Long = 30
+        refreshSecondsBeforeTTL: Long = 30,
```

Comment:
I don't think this name is accurate, and it's not descriptive. "refreshSlackTimeSeconds" might be closer to what it does, but is also not a good name (it would need a comment to explain that).

There's no "time to live" here, so TTL is wrong... but if we do keep "TTL" it should be "Ttl", we only capitalize the first letter of acronyms in variable names.

Reply:
as discussed we pick `refreshSecondsBeforeExpiration` for it 😁